libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ActivityUtils.java

```diff
@@ -31,4 +31,15 @@ public static void hideKeyboardForced(@Nullable final View view) {
                 .getSystemService(Context.INPUT_METHOD_SERVICE);
         inputMethodManager.hideSoftInputFromWindow(view.getWindowToken(), 0);
     }
+
+    /**
+     * Shows the keyboard for the given {@link View} using the {@link InputMethodManager#SHOW_IMPLICIT} flag,
+     * which is an implicit request (i.e. not requested by the user) to show the keyboard.
+     */
+    public static void showKeyboard(@Nullable final View view) {
```

Comment:
There's the `EditTextUtils.showSoftInput()` method that seems to do the same task of opening the keyboard. Can we reuse that one perhaps?

Reply:
The `showKeyboard` method was added to `ActivityUtils` since its counterparts, `hideKeyboard` and `hideKeyboardForced`, live there.  I updated the show and hide methods in `LoginEmailFragment` to use `EditTextUtils` in be564a278d2250392fe09f8e728849f62d13f2d9.  We should consolidate similar methods in `ActivityUtils` and `EditTextUtils` at some point.  Those changes might be better made outside of this pull request since they will affect files irrelevant to the email hints feature.

Reply:
Thanks for the change

Now that `ActivityUtils.showKeyboard()` is not used in this PR, can we completely remove it from the PR?

> We should consolidate similar methods in ActivityUtils and EditTextUtils at some point. Those changes might be better made outside of this pull request

Agree. We should consolidate them and do that in a separate PR. I opened #6940 to track that.

Reply:
Even though `ActivityUtils.showKeyboard` is no longer used in this pull request, I think we should make an exception and keep the changes since we're more likely to move the relevant methods from the `EditTextUtils` class to the `ActivityUtils` class in the `WordPressUtils` library for use elsewhere.  Keeping the changes will allow the library to include complementary methods and make the consolidation described in https://github.com/wordpress-mobile/WordPress-Android/issues/6940 easier.

Reply:
I see. Then please, let's add an in-code comment (`// TODO:` perhaps) in `EditTextUtils`'s version to state exactly that, that the method should be retired in favor of `ActivityUtils`'s one and with a small reasoning. Anyone reading the code without such a comment will be puzzled which one to use otherwise, especially since EditTextUtils's has lots of use sites while ActivityUtils's has none. Thanks!

Reply:
I added the comments in f24314a0a42053ef06a6fc455428bdd2a8192acc.